P5/Source/Specs/att.datcat.xml

```diff
@@ -219,6 +270,48 @@ $Id$
       its values, which are used as direct references to data categories; hence the use of
         <att>datcat</att> in the <gi>symbol</gi> element.</p>
   </exemplum>
+  <exemplum xml:lang="en">
+    <p>The <ident type="class">att.datcat</ident> attributes can be used for any sort of taxonomies. The example below illustrates their usefulness for describing usage domain labels in dictionaries on the example of Morais (REFERENCE). </p>
```

Comment:
@bansp Council is reviewing this here at Paderborn, and noticed that we're missing a reference here--presumably something to add to BIB, too. Can you add these?

Reply:
Thanks, @ebeshero . I left that for the Lex-0 colleagues to add, and with at least @ttasovac and (I hope) @laurentromary present in Paderborn, this is going to be a quick job.

Reply:
@ebeshero Please kindly review the most recent edit. I've made it online, so without any checks beyond those offered by Github. I hope hi/@rend="it" was the right markup to use.

Reply:
I've backported the edit to the LingSIG repo and the result is going to be soon shown under 
https://jenkins-paderborn.tei-c.org/view/LingSIG/job/TEIP5-LingSIG-tests/lastSuccessfulBuild/artifact/P5/release/doc/tei-p5-doc/en/html/ref-att.datcat.html